nms/app/packages/magmalte/app/views/equipment/FEGEquipmentGatewayKPIs.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {DataRows} from '../../components/DataGrid';
+
+import DataGrid from '../../components/DataGrid';
+import FEGGatewayContext from '../../components/context/FEGGatewayContext';
+import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
+import React from 'react';
+import nullthrows from '@fbcnms/util/nullthrows';
+import useMagmaAPI from '@fbcnms/ui/magma/useMagmaAPI';
+
+import {HEALTHY_STATUS, UNHEALTHY_STATUS} from '../../components/GatewayUtils';
+import {getLatency} from './EquipmentGatewayKPIs';
+import {useContext} from 'react';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+/**
+ * Displays the maximum latency, minimum latency, average latency,
+ * total federation gateway count, healthy federation gateway count,
+ * and the percentage of healthy federation gateways.
+ */
+export default function FEGEquipmentGatewayKPIs() {
+  const {match} = useRouter();
+  const ctx = useContext(FEGGatewayContext);
+  const fegGatewaysHealthStatus = ctx.health;
+  const networkId: string = nullthrows(match.params.networkId);
+  const timeRange = '3h';
+  const {response: maxResponse} = useMagmaAPI(
+    MagmaV1API.getNetworksByNetworkIdPrometheusQuery,
+    {
+      networkId: networkId,
+      query: `max_over_time(magmad_ping_rtt_ms{service="magmad",metric="rtt_ms"}[${timeRange}])`,
+    },
+  );
+
+  const {response: minResponse} = useMagmaAPI(
+    MagmaV1API.getNetworksByNetworkIdPrometheusQuery,
+    {
+      networkId: networkId,
+      query: `min_over_time(magmad_ping_rtt_ms{service="magmad",metric="rtt_ms"}[${timeRange}])`,
+    },
+  );
+
+  const {response: avgResponse} = useMagmaAPI(
+    MagmaV1API.getNetworksByNetworkIdPrometheusQuery,
+    {
+      networkId: networkId,
+      query: `avg_over_time(magmad_ping_rtt_ms{service="magmad",metric="rtt_ms"}[${timeRange}])`,
+    },
+  );
+
+  const maxLatency = getLatency(maxResponse, Math.max);
+  const minLatency = getLatency(minResponse, Math.min);
+  const avgLatencyArr = avgResponse?.data?.result
```

Comment:
As opposed to adding a suffix of `Arr` to denote its type, add a type annotation, and change the name to something meaningful like `avgLatencies`